app/distros/octane.js

```diff
@@ -0,0 +1,33 @@
+var request = require('request');
+
+function init() {
+  var module = {
+    id: 'octane',
+    profile: 'lightning'
```

Comment:
Nope, this is correct for now.  Right now Octane is just a composer.json and doesn't have its own profile.  Eventually this will be changed but for now Lightning is actually the installed profile (Octane is just adding modules)